core/blockchain.go

```diff
@@ -166,14 +167,15 @@ type CacheConfig struct {
 }
 
 var DefaultCacheConfig = &CacheConfig{
```

Comment:
Is there a reason we don't align these with the values in `./plugin/evm/config.go`?